application/common/helpers/Utils.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace common\helpers;
+
+class Utils
```

Comment:
What would you think of making this a more meaningful class (e.g. `Time`, `UtcTime`, or similar)? I ask because I read (or heard) someone articulate that a Utils class is itself a code smell, and that seems accurate to me.